010-OFEP-flagd-project-charter.md

```diff
@@ -0,0 +1,41 @@
+## Make flagD a full sub-project of OpenFeature
+
+## State: PENDING REVIEW
+
+
+This OFEP proposes to make flagD a full subproject of OpenFeature. The project will get a dedicated website, documentation, roadmap and charter. 
+
+## Background
+
+flagD started as small reference implementation for a developer-focussed feature evaluation mechanism, enabling OpenFeature SDK users to experiment with an end-to-end solution. Over time the project has grown in functionality and become stable and is now also used in production settings. 
+
+In the past there was also some misunderstanding of the relationship of OpenFeature and flagD. 
+
+Therefore, the project should become a full sub-project of open feature. 
+
+## Proposal
+
+flagD will become a dedicated sub project of the OpenFeature project. Documentation for flagD and all examples for OpenFeature will be moved to a dedicted space under the newly created domain ``openfeature.dev``.
+
+The sub project gets it's own dedicated charter to ensure the project has clearly defined goals and scope. As a sub project flagD will have its own maintainers but will still be under the governance of the OpenFeature project. 
+
+Below is the charter for flagD as defined by the OpenFeature governance board. 
+
+
+### Charter
+
+FlagD is an open-source project which provides a portable, lightweight, production-ready, and OpenFeature-compliant feature flag evaluation daemon, along with a supporting k8s operator and set of OpenFeature SDK providers.
+
+The flagD project aims to provide a developer-focused, cloud-native, lightweight and extensible feature evaluation engine.
```

Comment:
should we still highlight the fact that flagd will continue to be a reference implementation for OpenFeature ? Or is this no longer a goal?

Reply:
It is a full, production ready implementation not _just_ a reference implementation.